website/docs/docs/core/connect-data-platform/fabric-setup.md

```diff
@@ -84,14 +84,14 @@ More details about how these values affect your connection and how they are used
 
 SQL Server and windows authentication are not supported by Microsoft Fabric Synapse Data Warehouse.
 
-### Azure Active Directory Authentication (AAD)
+### Microsoft Entra ID authentication
 
-Azure Active Directory authentication is a default authentication mechanism in Microsoft Fabric Synapse Data Warehouse.
+Microsoft Entra ID (formerly Azure AD) authentication is a default authentication mechanism in Microsoft Fabric Synapse Data Warehouse.
```

Comment:
since "Microsoft Entra ID authentication" has a long character length (for page ToC) went with adding "formerly Azure AD" in the paragraph instead of title